var/spack/repos/builtin/packages/py-kb-python/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyKbPython(PythonPackage):
+    """Python wrapper around kallisto | bustools for scRNA-seq analysis."""
+
+    homepage = "https://github.com/pachterlab/kb_python"
+    pypi = "kb_python/kb_python-0.27.3.tar.gz"
+
+    version("0.27.3", sha256="dc98f6ceb4402d666b7e0d19be17c63d33e8b710a35cdc33de7c0f457122f43f")
+
+    depends_on("python@3.6:", type=("build", "run"))
+
+    depends_on("py-setuptools", type="build")
+
+    depends_on("py-anndata@0.6.22.post1:", type=("build", "run"))
+    depends_on("py-h5py@2.10.0:", type=("build", "run"))
+    depends_on("py-jinja2@2.10.1:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-jinja2@2.10.2:", type=("build", "run"))
```

Reply:
Good catch. I read the Jinja2>2.10.1 dependency and "filled-in" the ">=".